pkg/cmd/convert.go

```diff
@@ -62,15 +69,39 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
```

Comment:
@containscafeine you don't use `data` outside of that for. declare it before the first `if` inside `for`